lib/cluster/rds/event_subscription_creator.rb

```diff
@@ -0,0 +1,38 @@
+module Cluster
+  class RDS::EventSubscriptionCreator < Base
+    def self.create
+      db_instance_identifier = rds_name
+      if subscription_exists?
```

Comment:
all this is necessary because the creation of event subscriptions is not idempotent. Odd, given that most other "alert me when things happen" services in AWS are.